File: backend/models/answer.py
from pydantic import BaseModel
from models.author import Author


class Answer(BaseModel):
    aId: str = None
    description: str = ""
    createdTime: int = None
    updatedTime: int = None
    commentCount: int = None
    voteupCount: int = None
    author: Author = None
    content: str

File: frontend/lib/models/answer.dart
import 'package:simple_forum_app/models/author.dart';
import 'package:simple_forum_app/models/item.dart';

class Answer implements Item{
  String aId;
  String description;
  int createdTime;
  int updatedTime;
  int commentCount;
  int voteupCount;
  Author author;
  String content;

    Answer.blank()
      : aId = "",
        content = "",
        author = Author.blank();

 Answer.fromJson(Map<String, dynamic> json) {
   
      aId = json["aId"];
      description = json["description"];
      createdTime = json["createdTime"];
      updatedTime = json["updtedTime"];
      commentCount = json["commentCount"];
      voteupCount = json["voteupCount"];
      content = json["content"];


      if (json["author"] != null) {
        author = Author.fromJson(json["author"]);
      }

  }
}